.github/workflows/ci-python.yml

```diff
@@ -1,4 +1,4 @@
-name: ci-rust
+name: CI Python
```

Comment:
Name should be `ci-python` to be consistent with the other workflows

Reply:
done